### PR TITLE
Bug 1212864 - Remove homescreen pinch gesture. r=gasolin,gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=892306868c9882ec40782659ea17761fae4f4293

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90a990eeccec45251d5e81fae0c28d10f1a15cc3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=317816c4d7b72325e79cd094695edced36fca8aa

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=06d9534e94a8bda1ce457719551b060e87137576

File: apps/homescreen/js/app.js [hunk 1]
```diff
@@ -364,6 +357,46 @@ const SETTINGS_VERSION = 0;
...
+        stores[0].addEventListener('change', e => {
+          if (e.id === COLUMNS_SETTING) {
+            syncSmallSetting();
+          }
+        });
+        syncSmallSetting();
+      }, e => {
+        console.error('Error retrieving home screen settings datastore:', e);
+      });
+    },
```
Comment (gmarty) [inline apps/homescreen/js/app.js]: This seems to harm readability. I know the logic is simple but maybe you can move this section to its own script (like datastore.js and pagesstore.js).

Comment (Cwiiis) [inline apps/homescreen/js/app.js]: That sounds reasonable - have a settings.js...

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8ded5341c2b0ac8d50556ff49f89bbc051b1ff4d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2f6c10641a2ae10869e918f8b5d9ac8c788a4100

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f786fb7b1a6aac79f44bc36108512e863657a69e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=db1ca616bfb6008b627ed63f0063a711b0abf1ff

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9ed2ae8b3592ceed6eeaf91630fa8bfef8b99244

**Pull request merged**